src/styles/blocks/card.scss

```
@@ -0,0 +1,148 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: $card-width;
+  height: 408px;
+  padding-top: 32px;
+
+  border: 1px solid $border-color;
+  transition: transform 0.3s ease-in-out;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__container {
+    padding: 0 16px 16px;
+  }
+
+  &__img {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+
+    img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 1.5;
+    font-weight: 500;
+    color: $main-color;
+
+    margin-bottom: 4px;
+
+    transition: color 0.3s ease-in-out;
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    color: $secondary-color;
+
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    color: $main-color;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 1.5;
+    color: $secondary-color;
+  }
+
+  &__amount {
+    font-size: 16px;
+    line-height: 1.13;
+    font-weight: 700;
+    color: $main-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    font-size: 14px;
+    font-weight: 700;
+    color: #fff;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    background-color: $primary-color;
+
+    border-radius: 5px;
+    border: 1px solid $primary-color;
+
+    height: 40px;
+
+    transition: all 0.3s ease-in-out;
```

```suggestion
    transition: background-color, color 0.3s ease-in-out;
```
Don't use the 'all' keyword in CSS transitions, it's bad for performance
[Here](https://www.pno.dev/articles/dont-use-the-all-keyword-in-css-transitions/) you can read more about it